File: Dataset/Leetcode/test/94/321.java
 class Solution {
    public List<Integer> XXX(TreeNode root) {
        List<Integer> list = new ArrayList<>();
        helper(root, list);
        return list;
    }

    public void helper(TreeNode root, List list){
        if(null == root){
            return;
        }
        helper(root.left, list);
        list.add(root.val);
        helper(root.right, list);
    }
}

File: Dataset/Leetcode/train/7/321.py
class Solution:
    def XXX(self, x: int) -> int:
        s = str(x)
        s = "-" + s.replace("-","")[::-1] if "-" in s else s[::-1]
        return (int(s) if int(s)<2**31-1 and int(s)>-2**31 else 0)

